feat(render): add canvas width/height helpers

gabe565 opened the pull request:
Adds a new `canvas` variable to `render.star`. The `canvas` var has helpers `canvas.width()` and `canvas.height()`.

Example:
```starlark
load("render.star", "render", "canvas")

def main(config):
    w, h = canvas.width(), canvas.height()
    return render.Root(
        child = render.Padding(
            child = render.Text("%dx%d" % (w, h)),
            pad = 1,
        ),
    )
```

gabe565 commented:
@IngmarStein Actually, do you think something like `render.width()` would make more sense? If so, I could move this logic into the render module.

IngmarStein commented:
Good point. Pixlet doesn't _actually_ get to know anything about a device, just a canvas size and rendering options. How about `render.canvas_width()`, to be explicit which `width` this refers to?

gabe565 commented:
@IngmarStein I changed it again. I realized modules can provide multiple variables. So now, `render.star` can also provide `canvas`. I updated the initial PR description, but basically an app can now do `canvas.width()` and `canvas.height()`

No other modules behave this way, but it's fully supported by Skylark. If you think this is too weird, I could just go back to the original code and rename `device.star` to `canvas.star`.
